conda/recipes/dask-cuda/meta.yaml

```diff
@@ -23,12 +23,9 @@ requirements:
     - python
     - setuptools
   run:
-    - python
-    - dask >=2.4.0
-    - distributed >=2.18.0
-    - pynvml >=8.0.3
-    - numpy >=1.16.0
-    - numba >=0.50.0,!=0.51.0
+  {% for package in data.get('packages') %}
```

Comment:
Should we also be using [`resolved_packages('host')`](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html?highlight=load_setup_py_data#conda-build-specific-jinja2-functions) here to include the host packages?

Reply:
We probably want to use something like [`data.get('install_requires', [])`]( https://github.com/conda/conda-build/blob/3b99b2222a067e113a2282926871cd1e5406ee2b/conda_build/templates/setuptools.yaml#L28-L30 )

`resolved_packages` pins down to the exact package used at build time, which isn't likely what we want

Reply:
That makes sense, but in that case should we update `requirements.txt`? It looks like it has different version restrictions from the current recipe - for example:

https://github.com/rapidsai/dask-cuda/blob/3c6e0a9d18615376c05074622b0cec2fc1189aeb/requirements.txt#L5

https://github.com/rapidsai/dask-cuda/blob/3c6e0a9d18615376c05074622b0cec2fc1189aeb/conda/recipes/dask-cuda/meta.yaml#L31

Reply:
Yeah I think so. Thanks for catching that 🙂